migrations/versions/600_add_units_to_lots.py

```diff
@@ -0,0 +1,38 @@
+"""Add units to lots, for display by the frontend aps
+
+Revision ID: 600
+Revises: 590
+Create Date: 2016-05-05 10:49:45.258601
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '600'
+down_revision = '590'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+def upgrade():
+    op.add_column('lots', sa.Column('data', postgresql.JSON(), nullable=True))
+    op.execute("""
+        UPDATE lots
+          SET data = '{"unitSingular": "service", "unitPlural": "services"}'
+          WHERE  slug in ('saas', 'paas', 'iaas', 'scs', 'digital-outcomes', 'user-research-participants');
+    """)
+    op.execute("""
+        UPDATE lots
+          SET data = '{"unitSingular": "specialist", "unitPlural": "specialists"}'
```

Comment:
hmm, yes I think you're right.  I was thinking of the frameworks questions where we spilt into multiple questions on the basis of `specialist` vs. `service`, but maybe where this will be used `service` is still the thing.

I've also spotted we have things like `{{ "this digital specialist role" if lot == 'digital-specialists' else "a service in this category" }}`

Beginning to think this isn't going to be the end of our problems by any means.